apiary.apib

```diff
@@ -1612,14 +1612,17 @@ Removes the specified preference from the user's record
 + Response 204 (application/json)
 
 
-## User Organization List [/users/{user_id}/organizations]
+## User Organization List [/users/organizations/{?username}]
 
 It allows clients to retrieve a list of organizations a user belongs to
 
 ### User Organization List [GET]
 
 Returns a JSON representation of the paginated list of organizations for a user
 
++ Parameters
+    + username (required)
```

Comment:
This is actually optional, not required. You could say "(staff permission required for users other than self)"